JayBhatt25/my-Portfolio
# fix: projects section centering issue

###  What does this PR do?

Fixed the projects section centering issue caused by the upward button that is acting as a see less button

Fixes #1 

###  Screenshots/Videos

[protfolio-projects-centering-issue-fixed.webm](https://github.com/JayBhatt25/my-Portfolio/assets/73158584/499b2ab7-f8c7-4721-a045-97564aef176a)

### Type of change

Bug fix (non-breaking change which fixes an issue)

### How should it be tested?

1. Go to the Index Page
2. Click on Projects link in the navbar
3. Click on see more downward button until the see less button is visible
4. Click on see less upward button
5. Verify if the screen is centered on the Projects section after clicking see less button
